android/build.gradle

```diff
@@ -4,7 +4,7 @@ def DEFAULT_COMPILE_SDK_VERSION             = 28
 def DEFAULT_BUILD_TOOLS_VERSION             = "28.0.3"
 def DEFAULT_MIN_SDK_VERSION                 = 16
 def DEFAULT_TARGET_SDK_VERSION              = 28
-def DEFAULT_FACEBOOK_SDK_VERSION            = "4.38.1"
+def DEFAULT_FACEBOOK_SDK_VERSION            = "5.0.0"
```

Comment:
@janicduplessis wouldn't it be better to set this to `[5.0.0,5.1[` so we support 5.0.x? I think there shouldn't be any breaking changes in that range. i've set this to `~> 5.0.0` on iOS which is the same range for the same reason